src/main/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConfig.java

```diff
@@ -124,6 +129,11 @@ public class MarkLogicSinkConfig extends AbstractConfig {
         .define(DMSDK_INCLUDE_KAFKA_METADATA, Type.BOOLEAN, null, Importance.LOW,
             "Set to true so that Kafka record metadata is added to document metadata before it is written. If the document fails to be written, the Kafka record metadata will be logged as well.")
 
+        // TODO Need more info here on the API declaration itself?
```

Comment:
I am going to resolve this before the 1.7.0 release by having it point to content in the README, just haven't written that content yet.